composer.json

```diff
@@ -55,6 +55,9 @@
         "saltybeagle/savvy_turbo": "0.1.*",
         "phpunit/phpunit": "~4.3",
         "ezyang/htmlpurifier": "~4.6",
-        "guzzlehttp/guzzle": "~5.1"
+        "guzzlehttp/guzzle": "^6.0",
+        "unl/rev-api": "0.1.2-beta",
+        "phansys/getid3": " ~2.1@dev",
+        "swiftmailer/swiftmailer": "@stable"
```

Comment:
I would encourage you to add the php version requirement so composer will warn people when they try to install on an unsupported platform:

```
"php": ">=5.6.0"
```